docs/install/dns-setup.md

```diff
@@ -0,0 +1,157 @@
+# DNS Setup
+
+When running FlowForge on Docker or Kubernetes you will need to be able to setup an entry in a DNS server.
+
+This is because FlowForge uses hostname based routing to know which project you want to access.
+
+By default the project name is used in combination with a supplied domain. In this document I will use `example.com` as the domain. (It doesn't need to be a "whole" domain, it could also be a sub domain of one you already own. e.g. `ff.example.com`).
+
+If you are running Docker/Kubernetes on the same machine as the DNS server and Web Browser do not use `127.0.0.1` as the IP address to point the wild card domain at. This is because the host names will be looked up by the Web Browser, the Forge Application, and the Project Node-RED instances. The last 2 are running in containers and `127.0.0.1` will resolve to the container, not the entry point. 
+
+## Production
+
+For a production deployment you will need to talk to who ever owns your DNS infrastructure.
```

Comment:
I'd assume most customers (with a domain) would CRUD their DNS via a web based control panel so will set up the DNS record themselves. My point is, I think we should just tell them what they need to set up rather than suggest they seek help.

Reply:
We do in the next 2 sections, but we don't know if it's going to be a A record or a CNAME record until we know where/how they have deployed.

Reply:
Sorry if it's not clear, I'm saying that this sentence implies the reader can't set up the DNS records themselves but I'd expect in most cases they can. If I could remember how to suggest an edit I would say it should be changed to...

For a production deployment you will need to have access to your DNS control panel, if you are not sure how to set up DNS records talk to whoever manages your DNS.